### DISCUSSION TITLE
Can we connect to Facebook shop

### DISCUSSION THREAD

**Discussion by marifrahman** [bagisto/bagisto#9784] [General]
Hi,
I am new to this project. Just wondering if there is any plugin to connect multi vendors to their respective Facebook shops ?

**Comment by amit-webkul** [comment]
Hello, @marifrahman 

In Bagisto, we don't have a connector that directly connects your store to Facebook Shop. If you need to connect your Bagisto store to your Facebook shop, you can purchase a customization for this.
